ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -82,6 +82,10 @@ message CreatedEvent {
   // This means a newer client works with an older sandbox seamlessly.
   // Optional
   google.protobuf.StringValue agreement_text = 6;
+
+  // Metadata of the contract. Required for contracts created
+  // after the introduction of this feature.
+  ContractMetadata metadata = 10;
```

Comment:
How big can this field be? Does it make sense to have a toggle on the subscription request to disable populating it if the client doesn't care about it? CC @mziolekda

Reply:
The timestamp and the key hash are "small" and constant size. The driver metadata is AFAIK empty on sandbox-on-x and VMBC. I'm not sure about its size on Canton, best would be to ask the corresponding teams.

Reply:
Thanks @rautenrieth-da . @daravep Can you maybe shed some light on the expectations of the driver metadata size for Canton? Is it arbitrarily big?

Reply:
I'd be very reluctant to commit to an upper size right now. We currently plan to include the following information in there in the near future:

* A cryptographic nonce for authenticated contract IDs (essentially a SHA256 hash)
* The guessed residence domain of the contracts (a Canton unique identifier, i.e., 256 bytes)

Depending on what comes out of the multi-domain design work, we may also want to include causality information in there. We currently do not have any estimates how much data this will be, but it is likely to be unbounded (linear in the number of domains that exist).

Reply:
Thanks for the clarification @andreaslochbihler-da . The question now is: do we want to have the possibility to disable forwarding the driverMetadata in the transaction streams? This is mostly geared towards Canton, since the other ledger implementations do not populate such a field